media-converter/src/videoconv.rs

```diff
@@ -216,10 +216,7 @@ impl<'a> Read for PadReader<'a> {
         if to_copy == out.len() {
             out.copy_from_slice(&self.chunk[self.chunk_offs..(self.chunk_offs + to_copy)]);
         }else{
-            /* FIXME: there's probably some cool functional way to do this */
-            for i in 0..to_copy {
-                out[i] = self.chunk[self.chunk_offs + i];
-            }
+            (0..to_copy).for_each(|i| out[i] = self.chunk[self.chunk_offs + i]);
```

Comment:
I think the whole `if` can be replaced with just `out[..to_copy].copy_from_slice(&self.chunk[self.chunk_offs..(self.chunk_offs + to_copy)]);` 😄

Reply:
done👍